examples/build-eth-block/main.go

```diff
@@ -64,22 +70,72 @@ func main() {
 		_ = bundleContract.SendConfidentialRequest("newBundle", newBundleArgs, confidentialDataBytes)
 	}
 
+	var blockBidID any
 	{ // Signal to the builder that it's time to build a new block
 		payloadArgsTuple := types.BuildBlockArgs{
 			ProposerPubkey: []byte{0x42},
-			Timestamp:      targetBlock + 12, //  ethHead + uint64(12),
+			Timestamp:      targetBlock.Time() + 12, //  ethHead + uint64(12),
 			FeeRecipient:   common.Address{0x42},
 		}
 
-		_ = ethBlockContract.SendConfidentialRequest("buildFromPool", []any{payloadArgsTuple, targetBlock + 1}, nil)
+		receipt := ethBlockContract.SendConfidentialRequest("buildFromPool", []any{payloadArgsTuple, targetBlock.NumberU64() + 1}, nil)
+		maybe(err)
+
+		for _, receiptLog := range receipt.Logs {
+			/// debug stuff ;; free to remove
+			logJSON, err := receiptLog.MarshalJSON()
+			maybe(err)
+			log.Printf("receipt log: %s", string(logJSON))
+			////////////////////////////////////////////////
+
+			if receiptLog.Topics[0] == ethBlockContract.Abi.Events["BuilderBoostBidEvent"].ID {
+				bids, err := ethBlockContract.Abi.Events["BuilderBoostBidEvent"].Inputs.Unpack(receiptLog.Data)
+				maybe(err)
+				blockBidID = bids[0] // the one we want for the merged-bundles data records
+			}
+		}
+	}
+
+	{ // Submit block to the relay
+		// get latest block from live chain
+		// execBlock, err := fr.L1.RPC().BlockByNumber(context.Background(), nil)
+		maybe(err)
+		beaconBlock, beaconRoot, err := fr.L1Beacon.GetBlockHeader(nil)
```

Comment:
I try to keep the examples valid for both local deployment (local suave-geth) too. Check an example [here](https://github.com/flashbots/suapp-examples/blob/main/examples/app-ofa-private/main.go#L29). Could we have a flag that skips this part of the process if running in local? (Since I am assuming it would be too much to mock everything locally even with Kurtosis).